client/src/main/java/io/split/client/AlwaysReturnControlSplitClient.java

```diff
@@ -14,6 +15,8 @@
  */
 public class AlwaysReturnControlSplitClient implements SplitClient {
 
+    private static final SplitResult RESULT_CONTROL = new SplitResult(Treatments.CONTROL, null);
```

Comment:
When a configuration is not defined, should we return an empty map instead of null? It would simplify / remove NPEs

Reply:
`config` is a string.

Reply:
🤦‍♂️ Wow not sure how i missed that. Ok null seems fair then